Sources/Quick/QuickSpec.m

```diff
@@ -115,10 +115,23 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
     }
 
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
-                              example.name.qck_selectorName,
-                              sanitizedFileName,
-                              (long)example.callsite.line];
+    
+    NSString *originalName = example.name.qck_selectorName;
+    NSString *selectorName = originalName;
+    unsigned int i = 2;
```

Comment:
nit-pick: Very silly, but I'd prefer to use `NSUInteger` here. :stuck_out_tongue_closed_eyes: 

I prefer to always use `NSInteger` and `NSUInteger` over `int` and `unsigned int`. [There are some good reasons to do so](http://stackoverflow.com/a/4445199/679254), although those are probably not relevant here.